module2/test_calculator.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+class Calculator:
+    def add(self, x, y):
+        return x + y
+
+    def subtract(self, x, y):
+        return x - y
+
+    def multiply(self, x, y):
+        return x * y
+
+    def divide(self, x, y):
+        if y == 0:
+            return "Error! Division by zero."
+        return x / y
+
+class TestCalculator(unittest.TestCase):
+    def setUp(self):
+        self.calc = Calculator()
+
+    def test_add(self):
+        self.assertEqual(self.calc.add(2, 3), 5)
+        self.assertEqual(self.calc.add(-1, 1), 0)
```

Comment:
Each scenario should be separaet test function, so that it will more easy to debug incase of test failure in what scenario test was failed